### PR TITLE
Update Mars on Ray doc

### PR DESCRIPTION
**Pull request by fyrestone**
## What do these changes do?

- Update Mars on Ray doc.
- Compatible with sphinx 6.0

## Related issue number

Fixes #xxxx

## Check code requirements

- [ ] tests added / passed (if needed)
- [ ] Ensure all linting tests pass, see [here](https://docs.pymars.org/en/latest/development/contributing.html#check-code-styles) for how to run them

### CODE REVIEW

File: docs/source/installation/ray.rst [hunk 1]
```diff
@@ -44,27 +45,27 @@ Creating a Mars on Ray runtime in the Ray cluster and do the computing:
...
+    session = mars.new_ray_session(backend='ray')
 
 Connect to the created Mars on Ray runtime and do the computing:
 
 .. code-block:: python
 
     import mars
     import mars.tensor as mt
-    session = mars.new_ray_session(address="http://ip:port", session_id="abcd", default=True)
+    session = mars.new_ray_session(address="ray://<supervisor virtual address>", session_id="abcd", default=True)
```
Comment (yuyiming) [inline docs/source/installation/ray.rst:68]: What is the format of "supervisor virtual address"? You can explain it or give some examples.

File: README.rst [hunk 2]
```diff
@@ -244,10 +244,8 @@ Starting a new Mars on Ray runtime locally via:
 
 .. code-block:: python
 
-    import ray
-    ray.init()
     import mars
-    mars.new_ray_session(worker_num=2)
+    mars.new_session(backend='ray')
```
Comment (chaokunyang) [inline README.rst:248]: Maybe both oscar and ray task mode should be documented here?

Comment (fyrestone) [reply README.rst:248]: > 

Too many modes confuse users. So, we only recommend the Ray DAG mode.

File: docs/source/installation/ray.rst (hunk 1 above)
Comment (fyrestone) [reply docs/source/installation/ray.rst]: I refined the doc. Thanks.

File: docs/source/installation/ray.rst [hunk 3]
```diff
@@ -44,66 +45,73 @@ Creating a Mars on Ray runtime in the Ray cluster and do the computing:
...
-Create a Mars on Ray runtime independently in the Ray cluster:
+Start a Ray actor for Mars supervisor:
 
 .. code-block:: python
 
     import mars
-    import mars.tensor as mt
-    cluster = mars.new_cluster_in_ray(worker_num=2, worker_mem=2 * 1024 ** 3)
+    # Start a Ray actor for Mars supervisor.
+    session = mars.new_ray_session(backend='ray')
```
Comment (zhongchun) [inline docs/source/installation/ray.rst:60]: There are `mars.new_session(backend='ray')` and `mars.new_ray_session(backend='ray')` here which may confuse users. Maybe we should specifically explain their differences.

Comment (fyrestone) [reply docs/source/installation/ray.rst:60]: Good suggestion. Let me refine the doc.

**Review by yuyiming — APPROVED** [review]
LGTM

**Review by zhongchun — APPROVED** [review]
LGTM.

**Review by chaokunyang — APPROVED** [review]
LGTM

**Pull request merged**